modules/apigee_edge_apidocs/src/Form/ApiDocSettingsForm.php

```diff
@@ -41,13 +42,55 @@ public function getFormId() {
    * {@inheritdoc}
    */
   public function submitForm(array &$form, FormStateInterface $form_state) {
+    $entity_type = \Drupal::entityTypeManager()
+      ->getStorage('apidoc')
+      ->getEntityType();
+    $config = $this->configFactory()->getEditable('apigee_edge_apidocs.settings');
+
+    $options = $form_state->getValue('options');
+    $config->set('default_revision', (bool) $options['new_revision'])->save();
+
+    $args = [
```

Comment:
An unnecessary new variable, you can pass this directly to `$this->messenger()`

Reply:
Fixed.